docs/msbuild/common-msbuild-project-items.md

```diff
@@ -125,6 +125,14 @@ In [!INCLUDE[vstecmsbuild](../extensibility/internals/includes/vstecmsbuild_md.m
 | Visible | Optional boolean. Indicates whether to display the file in **Solution Explorer** in [!INCLUDE[vsprvs](../code-quality/includes/vsprvs_md.md)]. |
 | CopyToOutputDirectory | Optional string. Determines whether to copy the file to the output directory. Values are:<br /><br /> 1.  Never<br />2.  Always<br />3.  PreserveNewest |
 
+### AssemblyMetadata
+ Represents assembly attributes to be generated as `[AssemblyMetadata(key, value)]`.
+
+| Item metadata name | Description |
+|-----------------------| - |
+| Include | Becomes the first parameter (the key) in the `AssemblyMetadataAttribute` attribute constructor. |
```

Comment:
Should this be key as above?

Reply:
Nope, the MSBuild equivalent of a key is the `Include` attribute in items.